src/extensions/exception-autocapture/index.ts

```diff
@@ -98,13 +90,13 @@ export class ExceptionObserver {
         }
 
         try {
-            if (!this.unwrapOnError) {
+            if (!this.unwrapOnError && this.config.capture_unhandled_errors) {
                 this.unwrapOnError = wrapOnError(this.captureException.bind(this))
             }
-            if (!this.unwrapUnhandledRejection) {
+            if (!this.unwrapUnhandledRejection && this.config.capture_unhandled_rejections) {
                 this.unwrapUnhandledRejection = wrapUnhandledRejection(this.captureException.bind(this))
             }
-            if (!this.unwrapConsoleError) {
+            if (!this.unwrapConsoleError && this.config.capture_console_errors) {
```

Comment:
This is the key line. We shouldn't be wrapping unless it is explicitly enabled in the config